include/sway/config.h

```diff
@@ -41,6 +41,11 @@ enum binding_flags {
 	BINDING_TITLEBAR=16, // mouse only; trigger on container titlebar
 };
 
+enum keep_kbd_layout {
+	LAYOUT_GLOBAL,
+	LAYOUT_PER_WINDOW
```

Comment:
Can you please add an additional prefix such as `KEYBOARD_LAYOUT_GLOBAL`? Just to eliminate any possibility of people getting confused between keyboard layouts and container layouts.